authzed/api/v1/permission_service.proto

```diff
@@ -324,6 +328,9 @@ message LookupResourcesRequest {
 
   // subject is the subject with access to the resources.
   SubjectReference subject = 4 [ (validate.rules).message.required = true ];
+
+  /** context consists of named values that are injected into the caveat evaluation context **/
+  google.protobuf.Struct context = 5 [ (validate.rules).message.required = false ];
 }
 
 // LookupResourcesResponse contains a single matching resource object ID for the
```

Comment:
We need to add an enum here to indicate that the response is either `HAS_PERMISSION` or `MAYBE_PERMISSION`, along with the same "missing fields"